draft-ietf-rats-msg-wrap.md

```diff
@@ -93,12 +93,9 @@ entity:
 
 --- abstract
 
-This document defines the RATS conceptual message wrapper (CMW) format, a type of encapsulation format that can be used for any RATS messages, such as Evidence, Attestation Results, Endorsements, and Reference Values.
-Additionally, the document describes a collection type that enables the aggregation of one or more CMWs into a single message.
+This document defines a conceptual message wrapper (CMW) format - an encapsulation method applicable to any conceptual message, such as Evidence, Attestation Results, Endorsements, and Reference Values. It also describes a collection type that aggregates one or more CMWs into a single message.
```

Comment:
```suggestion
This document defines a conceptual message wrapper (CMW) format - an encapsulation method applicable to any conceptual message, such as Evidence, Attestation Results, Endorsements, and Reference Values. It also describes a collection type that envelopes one or more CMWs into a single aggreated message.
```

Reply:
done in [f7b2005](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/173/commits/f7b2005c0d6e6bd33291cde5af89fbab3a232d1f)